tidb-cloud/migrate-incremental-data-from-mysql.md

```diff
@@ -0,0 +1,245 @@
+
+mysql-instances:
+  - source-id: "mysql-replica-01"
+    block-allow-list:  "bw-rule-1"
+    route-rules: ["route-rule-1"]
+    filter-rules: ["tpcc-filter-rule"]
+    syncer-config-name: "global"                   # You can use the syncers incremental data configuration above.
+    meta:                                          # When task-mode is "incremental" and the target database does not have a checkpoint, DM uses the binlog position as the starting point. If the target database has a checkpoint, DM uses the checkpoint as the starting point.
+    binlog-name: "mysql-bin.000001"
+    binlog-pos: 77092852
+    binlog-gtid: "b631bcad-bb10-11ec-9eee-fec83cf2b903:1-640"
+
+## ******** Configuration of the Target TiDB on TiDB Cloud **********
+target-database:    # the Target TiDB on TiDB Cloud
+  host: "tidb.70593805.b973b556.ap-northeast-1.prod.aws.tidbcloud.com"
+  port: 4000
+  user: "root"
+  password: "oSWRLvR3F5GDIgm+l+9h3kB72VFWBUwzOw=="     # If the password is not empty, it is recommended to use a dmctl-encrypted cipher
+
+## ******** Function Configuration **********
+block-allow-list:
+  bw-rule-1:
+    do-dbs: ["~^tpcc.*"]
+
+routes:                       # Table renaming rules ('routes') from upstream to downstream tables, in order to support merging different tables into a single target table.
+  route-rule-1:               # Rule name.
+    schema-pattern: "tpcc"    # Rule for matching upstream schema names. It supports the wildcards "*" and "?".
+    target-schema: "tpdd"     # Name of the target schema.
+
+filters:
+  tpcc-filter-rule:
+    schema-pattern: "tpcc"
+    events: ["drop database"]
+    action: Ignore
+
+##  ******** Ignore check items **********
+ignore-checking-items: ["table_schema"]
+```
+
+For detailed task configurations, see [DM Task Configurations](https://docs.pingcap.com/tidb/stable/task-configuration-file-full).
+
+To run a data migration task smoothly, DM triggers a precheck automatically at the start of the task and returns the check results. DM starts the migration only after the precheck is passed. To trigger a precheck manually, run the check-task command.
+
+```shell
+[root@localhost ~]# tiup dmctl --master-addr 172.16.7.140:9261 check-task dm-task1.yaml
+tiup is checking updates for component dmctl ...
+Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 172.16.7.140:9261 check-task dm-task1.yaml
+{
+    "result": true,
+    "msg": "check pass!!!"
+}
+```
+
+## Step 4. Start the migration task
+
+Run the following command to start the migration task:
+
+```shell
+[root@localhost ~]# tiup dmctl --master-addr 172.16.7.140:9261 start-task dm-task1.yaml
```

Comment:
the filename is dm-task1.yaml here but `task.yaml` in Step 3. Please use one of them consistently.

Reply:
OK. Changed to dm-task1.yaml